assets/_scss/_navigation_additional-class.scss

```
@@ -168,7 +162,12 @@
 		.wp-block-navigation-item__content {
 			grid-row: 1 / 2;
 			grid-column: 1 / 3;
-			padding: 0.8em;
+			padding: var(--wp--custom--spacing--menu-indent);
+		}
+		// エディタで On click の時にコアが出力してくる padding:0 の上書き
```

Stylelint (17.11.1)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)